libs/wire-api-federation/src/Wire/API/Federation/API.hs

```diff
@@ -94,14 +99,31 @@ fedClient ::
 fedClient = clientIn (Proxy @api) (Proxy @m)
 
 fedQueueClient ::
-  forall (comp :: Component) (name :: Symbol) m api.
-  ( HasEmptyResponse api,
-    HasFedEndpoint comp api name,
-    HasClient m api,
-    m ~ FedQueueClient comp
+  forall tag api.
+  ( HasNotificationEndpoint tag,
+    -- api ~ NotificationAPI tag (NotificationComponent tag),
```

Comment:
When I include this constraint, I get an error about the non-injectivity of a type family and I don't know how to fix it.